docs/installation/octopus-in-container/octopus-server-container-linux.md

```diff
@@ -1,23 +1,21 @@
 ---
-title: Octopus Deploy Server Container Linux
+title: Octopus Deploy Server Linux Container
 description: An Octopus Deploy Server instance can be run directly from within a container.
 position: 1
 ---
 
 If you want to run an Octopus Deploy Windows container, please refer to the [Octopus Deploy Server Container Windows](/docs/installation/octopus-in-container/octopus-server-container-windows.md) documentation.
 
+**Octopus Deploy Linux Containers are part of our Early Access Program (EAP) and may contain bugs or be unstable.**
+
+Note: When using Linux containers on a Windows machine, please ensure you have [switched to Linux Containers](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers).
+
 Running the Octopus Deploy Server inside a container provides a simple way to set up an Octopus Deploy instance, and upgrading to the latest version of Octopus is just a matter of running a new container with the new image version.
 
 Although there are a few different configuration options, the following is a simple example of starting an Octopus Deploy server container:
 
 ```Bash
-$ docker run --interactive 
-             --detach
-             --name OctopusDeploy \
-             --publish 1322:8080 \
-             --env ACCEPT_EULA="Y" \
-             --env DB_CONNECTION_STRING="..." \
-             octopusdeploy/octopusdeploy:2019.13.4
+$ docker run --interactive --detach --name OctopusDeploy --publish 80:8080 --env ACCEPT_EULA="Y" --env DB_CONNECTION_STRING="..." octopusdeploy/octopusdeploy:2019.13.4
```

Comment:
i wonder if we can use @emilol's cool new partial to inject the latest version number here?

Reply:
we can! `!docker-image <octopusdeploy/octopusdeploy>` should do it #DocsPending